### DISCUSSION TITLE
Ongoing Rejections of Educational GitHub Application: Assistance Needed

### DISCUSSION THREAD

**Discussion by v1rushb** [community/community#59442] [GitHub Education]
### Hi everyone,

Dear GitHub Community,

I hope this message finds you all well. I've been facing an issue that I haven't been able to resolve and I am seeking your help.

I have been trying to apply for GitHub's education program. I've made around 22 failed attempts, each time ensuring I follow all the provided guidelines and prerequisites. Despite my efforts, my applications have consistently been rejected. I've scanned through many threads in this forum and tried implementing all the proposed solutions but, unfortunately, none have been effective so far.

Here are the error messages I have been encountering:

At first, I received the following:

"You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote. Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘Other’ option when submitting a GitHub Education support ticket."
"The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote."

Recently, these messages have been accompanied by:

"The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote."
"The GitHub account you are using was recently created, and you appear to be having difficulty demonstrating your academic affiliation. Please wait a few days then try again."
"You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote. Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘Other’ option when submitting a GitHub Education support ticket."
I have taken care to ensure that I meet all the eligibility requirements and have provided accurate information in my applications. I've double-checked my submissions for any potential errors, I've used a valid school-issued email, and I've supplied the necessary academic documents for verification. Still, I can't seem to get past these issues.

If anyone has encountered a similar problem, or can shed some light on potential solutions, I would greatly appreciate your help. I'd especially be grateful for advice on any less obvious issues that could be causing these persistent rejections.

I strongly believe in the value of GitHub for enhancing educational pursuits and I am eager to make the most of it during my academic journey. Your assistance in helping me overcome this hurdle would be most appreciated.

Thank you for taking the time to read my post and for any advice you can provide.

_No response_

**Comment by D7x7z49** [comment]
I use VPN network or Clash, but it's difficult for me to access Github. How should I solve it? Not at school during the vacation?
